roles/openafs_krbserver/tasks/main.yaml

```diff
@@ -4,7 +4,7 @@
 
 - name: CentOS 6 exceptions to the RH conventions
   include_vars: "centos6.yaml"
-  when: ansible_distribution == "CentOS" and ansible_distribution_major_version >= 6
+  when: ansible_distribution == "CentOS" and ansible_distribution_major_version >= "6"
```

Comment:
Thanks. (Actually, on my todo list is to fix task before this one and get rid of this extra unneeded task.  We should be using 'with_first_found' or some such. but i didnt have chance to try that yet, so i'll merge this first.)